test/client/submit.js

```diff
@@ -1210,5 +1212,189 @@ module.exports = function() {
         });
       });
     });
+
+    describe('metadata projection', function() {
```

Comment:
We've flexed metadata on ops that were broadcast over pubsub, but ops can also be fetched when submitting an op from a stale document (that isn't subscribed, and is behind the server); can we test that the ops we get in that case also have their metadata correctly set?

This would need us to tweak [`SubmitRequest.submit()`](https://github.com/share/sharedb/blob/cc0e3382bf3df4c38c7ccc6bc52da71aff8c9f74/lib/submit-request.js#L106).

**Also**, can we add a test case for an unsubscribed client calling fetch after another remote client has changed the doc: this is where you'll need the `backend._getSanitizedOps()` to handle metadata projection for us.